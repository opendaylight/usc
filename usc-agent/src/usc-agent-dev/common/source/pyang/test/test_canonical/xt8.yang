module xt8 {
  ext:foo bar;
  prefix "xt8";
  namespace "urn:uri:foo";   // expect namespace before prefix



  import ietf-yang-types { prefix yang; }
  import ext { prefix ext; }
  import ietf-yang-smiv2 { prefix smiv2; }
  contact "hej" { }
  description "arne" + /* hmm*/ ' ' + 'anka';
  organization "hopp";  // expected after import

  revision 2001-01-01 { description "2"; }
  revision 2000-01-11 { description "initinla"; }

  typedef myInt2 {
    status deprecated;  // expected after type
    type int8 {
      range "1 | 5..6";
    }
  }
}
  

