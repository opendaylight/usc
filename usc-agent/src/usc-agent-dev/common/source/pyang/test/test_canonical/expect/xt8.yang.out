xt8.yang:3: error: UNEXPECTED_KEYWORD_1
xt8.yang:4: error: UNEXPECTED_KEYWORD
xt8.yang:13: error: UNEXPECTED_KEYWORD
xt8.yang:19: error: UNEXPECTED_KEYWORD_1
xt8.yang:20: error: UNEXPECTED_KEYWORD
